### PR TITLE
PHPC-450: Fix test failures for MongoDB 3.2

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/PHPC-450

### CODE REVIEW

File: tests/server/server-executeQuery-005.phpt [hunk 1]
```diff
@@ -50,8 +50,10 @@ printf("Set profile level to 0 successfully: %s\n", (empty($result->ok) ? 'no' :
 --EXPECTF--
 Set profile level to 2 successfully: yes
 object(stdClass)#%d (%d) {
-  ["$query"]=>
-  object(stdClass)#%d (%d) {
+  ["find"]=>
+  string(%d) "%s"
+  ["filter"]=>
+  object(stdClass)#%d (1) {
```
Comment (bjori) [inline tests/server/server-executeQuery-005.phpt]: This looks like something that needs to be split into two tests.
One on pre 3.2, and one 3.2+

Comment (jmikola) [inline tests/server/server-executeQuery-005.phpt]: Good point. I'll split these and the fail point test to use the 3.0 test server.

File: tests/utils/tools.php [hunk 2]
```diff
@@ -391,10 +391,6 @@ function fromJSON($var) {
     return $func($var);
 }
 
-
-
-/* NOTE: Using this function will take down mongod ! */
```
Comment (bjori) [inline tests/utils/tools.php]: Is this no longer true?
It used to quite literally (and intentionally) result in an abort() to make sure the getmore failed completely

Comment (jmikola) [inline tests/utils/tools.php]: Oh, it's possible that getMore itself aborts (I don't recall offhand). I thought this was referring to the PHP function itself and sending the `configureFailPoint` command, which certainly wasn't true.

Comment (bjori) [inline tests/utils/tools.php]: No no, the comment is referring to `mongod`

Comment (jmikola) [inline tests/utils/tools.php]: Got it. I revised the comment below. :+1:

**Comment by derickr** [comment]
LGTM

**Pull request merged**